cni/network/network.go

```diff
@@ -777,7 +777,7 @@ func (plugin *NetPlugin) createEpInfo(opt *createEpInfoOpt) (*network.EndpointIn
 		EndpointID:  endpointID,
 		ContainerID: opt.args.ContainerID,
 		NetNsPath:   opt.args.Netns, // probably same value as epInfo.NetNs
-		IfName:      opt.args.IfName,
```

Comment:
instead can you declare local ifname variable and set it here: https://github.com/Azure/azure-container-networking/blob/a39d8c4c779f1d5130dd807f7b391d7c87250abd/cni/network/network.go#L757